server/routes/induction/common/qualificationDetailsController.ts

```diff
@@ -42,4 +45,24 @@ export default abstract class QualificationDetailsController extends InductionCo
     )
     return res.render('pages/induction/prePrisonEducation/qualificationDetails', { ...view.renderArgs })
   }
+
+  addQualificationToInductionDto(
```

Comment:
I've moved this method into the superclass here (from the Update class), so that both the Create and Update class can use it